src/main/java/uk/gov/hmcts/darts/arm/model/rpo/AbstractMatterResponse.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.darts.arm.model.rpo;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@NoArgsConstructor
+public abstract class AbstractMatterResponse {
```

Comment:
What is a MatterResponse? (Maybe worth updating the name to make it more clear)

Reply:
If you look at the ARM RPO json classes these are mostly named Matter. This is what ARM have named the steps involved in the RPO process.